ee/tabby-webserver/src/service/background_job/mod.rs

```diff
@@ -36,6 +37,43 @@ pub enum BackgroundJobEvent {
     SerializedBackgroundJob(String),
 }
 
+impl BackgroundJobEvent {
+    fn job_name(&self) -> Option<&'static str> {
+        match self {
+            BackgroundJobEvent::SchedulerGitRepository(_) => Some(SchedulerGitJob::NAME),
+            BackgroundJobEvent::SchedulerGithubGitlabRepository(_) => {
+                Some(SchedulerGithubGitlabJob::NAME)
+            }
+            BackgroundJobEvent::SyncThirdPartyRepositories(_) => Some(SyncIntegrationJob::NAME),
+            BackgroundJobEvent::WebCrawler(_) => Some(WebCrawlerJob::NAME),
+            BackgroundJobEvent::SerializedBackgroundJob(s) => {
+                let event = serde_json::from_str::<Self>(s);
+                event.ok().and_then(|event| event.job_name())
+            }
```

Comment:
there should never be case we wanna generate a job name for `SerializeBackgroundJob`. shall panic / generate an error.